adapters/resetdigital/resetdigital.go

```diff
@@ -0,0 +1,284 @@
+	if request.Site == nil {
+		return ""
+	}
+
+	return request.Site.Domain
+}
+
+func getCurrency(request *openrtb2.BidRequest) string {
+	if len(request.Cur) == 0 {
+		return "USD"
+	}
+
+	return request.Cur[0]
+}
+
+func (a *adapter) MakeRequests(requestData *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var (
+		requests []*adapters.RequestData
+		errors   []error
+	)
+
+	for i := range requestData.Imp {
+		imp := requestData.Imp[i]
+		bidType, err := getBidType(imp)
+		if err != nil {
+			errors = append(errors, err)
+			continue
+		}
+
+		splittedRequestData, err := processDataFromRequest(requestData, imp, bidType)
+		if err != nil {
+			errors = append(errors, err)
+			continue
+		}
+
+		requestBody, err := json.Marshal(splittedRequestData)
+		if err != nil {
+			errors = append(errors, err)
+			continue
+		}
+
+		requests = append(requests, &adapters.RequestData{
+			Method:  "POST",
+			Uri:     a.endpointUri,
+			Body:    requestBody,
+			Headers: getHeaders(requestData),
+		})
+	}
+
+	return requests, errors
+}
+
+func processDataFromRequest(requestData *openrtb2.BidRequest, imp openrtb2.Imp, bidType openrtb_ext.BidType) (resetDigitalRequest, error) {
+	var resetDigitalRequestData resetDigitalRequest
+
+	// Check if requestData.Site is not nil before accessing its fields
+	if requestData.Site != nil {
+		resetDigitalRequestData.Site.Domain = requestData.Site.Domain
+		resetDigitalRequestData.Site.Referrer = requestData.Site.Page
+	}
+
+	resetDigitalRequestData.Imps = append(resetDigitalRequestData.Imps, resetDigitalRequestImps{})
+	resetDigitalRequestData.Imps[0].BidID = requestData.ID
+	resetDigitalRequestData.Imps[0].ImpID = imp.ID
+
+	if bidType == openrtb_ext.BidTypeBanner {
+		resetDigitalRequestData.Imps[0].MediaTypes.Banner.Sizes = append(resetDigitalRequestData.Imps[0].MediaTypes.Banner.Sizes, []int64{imp.Banner.Format[0].W, imp.Banner.Format[0].H})
+	}
+	if bidType == openrtb_ext.BidTypeVideo {
+		resetDigitalRequestData.Imps[0].MediaTypes.Video.Sizes = append(resetDigitalRequestData.Imps[0].MediaTypes.Banner.Sizes, []int64{*imp.Video.W, *imp.Video.H})
+	}
+
+	var extData = make(map[string]interface{})
+	err := json.Unmarshal(imp.Ext, &extData)
+	if err != nil {
+		return resetDigitalRequestData, err
+	}
+
+	resetDigitalRequestData.Imps[0].ZoneID.PlacementID = extData["bidder"].(map[string]interface{})["placement_id"].(string)
+	if resetDigitalRequestData.Imps[0].ZoneID.PlacementID == "test" {
+		resetDigitalRequestData.Imps[0].ForceBid = true
+	}
+
+	return resetDigitalRequestData, nil
+}
+
+func (a *adapter) MakeBids(internalRequest *openrtb2.BidRequest, externalRequest *adapters.RequestData, response *adapters.ResponseData) (*adapters.BidderResponse, []error) {
+	if response.StatusCode == http.StatusNoContent || response.StatusCode == http.StatusBadRequest || response.StatusCode != http.StatusOK {
+		return nil, nil
+	}
```

Comment:
`http.StatusBadRequest` and  `non http.StatusOK` codes should not be excluded. Can be rewritten as below:

Make use of response utils for checking response status code. Refer following as example
https://github.com/prebid/prebid-server/blob/dd07431ef54c38a2617c04e76b3e7bda9c21f569/adapters/appnexus/appnexus.go#L159-L165